src/App.css

```diff
@@ -1,29 +1,85 @@
 .App {
-  text-align: center;
+  text-align:center;
+  -webkit-text-fill-color:black;
+  text-justify:auto;
+
+
+}
+
+
+input[type=text]{
+  width: auto;
+  padding:0%;
+  margin: 10px;
+  box-sizing: border-box;
+  border-radius: 4px;
+  border-color: red;
+  
 }
+input[type=email]{
```

Comment:
Remove [type=email] from this line and remove input[type="text"] and other input types

Reply:
@ArpanBatra01